.github/workflows/static-analysis-test.yaml

```
@@ -0,0 +1,33 @@
+name: Static analysis test
+on:
+  pull_request:
+    branches:
+      - master
+    # These hardcoded paths are set for the PoC on the Apache chart
+    paths:
+      - "bitnami/apache/templates/*"
+      - "bitnami/apache/values.yaml"
+
+jobs:
+  static-analysis-test:
+    name: Static analysis test
+    runs-on: ubuntu-20.04
```

Why not using latest as we do in the rest of workflows?
```suggestion
    runs-on: ubuntu-latest
```

---

The only concern would be if they stop adding Helm to the latest images, but it is highly unlikely so I agree we can use `latest` instead